cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -15,12 +15,25 @@ MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
 SHARED_DIRECTORY_COMPUTE="<%= @shared_directory_compute %>"
 SHARED_DIRECTORY_LOGIN="<%= @shared_directory_login %>"
+DISABLE_RESOURCE_CHECK=false
+
+while getopts "d" opt; do
+  case $opt in
+    d) DISABLE_RESOURCE_CHECK=true;;
+    *)
+      echo "Usage: $0 [-d]" >&2
+      exit 1
+      ;;
+  esac
+done
 
-# Check compute fleet status
-compute_fleet_status=$(get-compute-fleet-status.sh)
-if ! echo "$compute_fleet_status" | grep -q '"status": "STOPPED"'; then
-    echo "Compute fleet is not stopped. Please stop it before updating the munge key."
-    exit 1
+if ! $DISABLE_RESOURCE_CHECK; then
```

Comment:
This is implicitly using the [`true`](https://www.gnu.org/software/coreutils/manual/html_node/true-invocation.html#true-invocation) and [`false`](https://www.gnu.org/software/coreutils/manual/html_node/false-invocation.html#false-invocation). Are we sure this is solid enough?

Reply:
Since in today's discussion we don't have a better solution for now. And this PR needs to be merged ASAP. Let's leave it here and if we decide to modify it, we can open a new PR for it.